doc/api/events.md

```diff
@@ -1282,9 +1282,11 @@ This is not used in Node.js and is provided purely for completeness.
 added: v14.5.0
 -->
 
-* Type: {boolean} Always returns `false`.
+* Type: {boolean} Returns `false` for user constructed events and `true`
+  for Node.js internal events.
 
-This is not used in Node.js and is provided purely for completeness.
+Currently only `AbortSignal`s' `"abort"` event is fired with `isTrusted`
```

Comment:
```suggestion
Currently, only the `AbortSignal` `"abort"` event is fired with `isTrusted`
```

Reply:
"Currently" suggests that we expect that to change. Do we? If not, let's remove that word.

Reply:
I expect that to change.

Reply:
Namely, when/if we add APIs like `fetch` this is likely to change